docs/Gemfile

```diff
@@ -16,6 +16,9 @@ gem "minima", "~> 2.5"
 # If you have any plugins, put them here!
 group :jekyll_plugins do
   gem "jekyll-feed", "~> 0.12"
+  gem "webrick", "~> 1.7" # Ruby 3.0 doesn't include webrick anymore in 4.2.2 https://github.com/jekyll/jekyll/issues/8523
+  gem "down", "~> 1.1.0" # this is for jekyll-strapi integration down dep.
+  gem "jekyll-strapi", "~> 0.1.3" # must have gem installed to make use of jekyll-strapi integration
```

Comment:
Examples in "docs/" they don't need strapi nor jekyll-strapi to be installed - it is just pure jekyll...

Reply:
I think that webrick and down are not necessary for docs generating.

Reply:
please see comments on your repository. Webrick is the default server used when you launch a jekyll site...you are using python http if I remember correctly?